binding/rust/tests/porcupine_tests.rs

```diff
@@ -255,6 +255,46 @@ mod tests {
         );
     }
 
+    #[test]
```

Comment:
is there a rusty way to parameterized these tests? it helps with reducing the the # of lines we need to copy/paste after adding new languages; also the ultimate goal is to take the test data out of the test files and put it somewhere so that each binding can read it and generate necessary tests.

Reply:
No built in way but the defacto way is to use macros, I can go ahead and try that